### ISSUE TITLE
Decorative images should have an empty (but-present) alt-text

### ISSUE THREAD

**Issue by sigurdm** [dart-lang/pub-dev#7223]
Decorative images should have an empty (but-present) alt-text to avoid being picked up by screen readers

Examples on front page:
https://pub.dev/static/hash-64paj0g1/img/pub-dev-logo.svg
https://pub.dev/static/hash-64paj0g1/img/landing-01.webp
https://pub.dev/static/hash-64paj0g1/img/landing-02.webp
https://pub.dev/static/hash-64paj0g1/img/landing-03.webp
